crates/turbo-tasks-fs/Cargo.toml

```diff
@@ -23,7 +23,8 @@ include_dir = { version = "0.7.2", features = ["nightly"] }
 jsonc-parser = { version = "0.21.0", features = ["serde"] }
 lazy_static = "1.4.0"
 mime = "0.3.16"
-notify = "4.0.17"
+notify = { version = "5.0.0", features = ["macos_kqueue"] }
```

Comment:
Could we ship the PR without kqueue? notify v5 is probably still a good change

Reply:
Is there a downside?